_includes/components/image-example-correct.html

```diff
@@ -0,0 +1,24 @@
+{% comment %}
+include
+- image
+- text
+{% endcomment %}
+<div class="usa-image-example--correct">
+    <figure class="usa-image-example__figure">
+      <img class="usa-image-example__image" alt="correct example, use dark background" src="{{ include.image | prepend: site.baseurl}}" />
+      <figcaption class="usa-image-example__figcaption">
+        <ul class="usa-icon-list usa-icon-list--size-md">
+          <li class="usa-icon-list__item">
+            <div class="usa-icon-list__icon">
+              <svg class="usa-icon" aria-hidden="true" role="img">
+                <use xlink:href="/assets/img/sprite.svg#check_circle"></use>
```

Comment:
We'll want `{{ site.baseurl }}` here for the Cloud.gov Pages preview environments.

```suggestion
                <use xlink:href="{{ site.baseurl }}/assets/img/sprite.svg#check_circle"></use>
```